mozilla-mobile/android-components/components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,26 @@
     <!-- The text shown inside the advanced options button used to bypass the invalid SSL certificate. It's only shown if the user has expanded the advanced options. -->
     <string name="mozac_browser_errorpages_security_bad_cert_accept_temporary">Accept the Risk and Continue</string>
 
+    <!-- The document title and heading of the error page shown when a website uses HSTS. -->
+    <string name="mozac_browser_errorpages_security_bad_hsts_cert_title">This website requires a secure connection.</string>
+    <!-- The error message shown when a website uses HSTS. -->
+    <string name="mozac_browser_errorpages_security_bad_hsts_cert_message"><![CDATA[
+      <ul>
+         <li>The page you are trying to view cannot be shown because this website requires a secure connection.</li>
+         <li>The issue is most likely with the website, and there is nothing you can do to resolve it.</li>
+         <li>You can notify the website’s administrator about the problem.</li>
+      </ul>
+    ]]></string>
+
+    <!-- The text shown inside the advanced button used to expand the advanced options. It's only shown when a website uses HSTS. -->
+    <string name="mozac_browser_errorpages_security_bad_hsts_cert_advanced">Advanced…</string>
+    <!-- The advanced certificate information shown when a website uses HSTS. The %1$s will be replaced by the website URL and %2$s will be replaced by the app name. It's only shown when a website uses HSTS. -->
+    <string name="mozac_browser_errorpages_security_bad_hsts_cert_techInfo"><![CDATA[
+        <label> <b>%1$s</b> has a security policy called HTTP Strict Transport Security (HSTS), which means that <b>%2$s</b> can only connect to it securely. You can’t add an exception to visit this site.
```

Comment:
@mcarare 
The string `mozac_browser_errorpages_security_bad_hsts_cert_techInfo` is missing a closing `</label>` tag. 

Could you update this with a new ID when you make the fix, since this has already gone out to localizers and if we don't change the ID they won't be notified of the text change.